open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/IconRegistryImpl.java

```diff
@@ -474,12 +483,31 @@ public boolean removeIcon(final int iconId, final Object source)
             myChangeSupport.notifyListeners(listener -> listener.iconsRemoved(Collections.singletonList(fRemoved), source),
                     EXECUTOR);
             saveLater();
+            // updateCollectionNames();

```

Comment:
This method and call are both commented

Reply:
Nah, it's obsolete. Removed (along with the method)